plugins/discourse-calendar/app/views/discourse_post_event/events/index.ics.erb

```diff
@@ -0,0 +1,15 @@
+BEGIN:VCALENDAR
+VERSION:2.0
+PRODID:-//Discourse//<%= Discourse.current_hostname %>//<%= Discourse.full_version %>//EN
+<% @events.each do |event| %>
+BEGIN:VEVENT
+UID:calendar_event_#<%= event.id %>@<%= Discourse.current_hostname %>
+DTSTAMP:<%= Time.now.utc.strftime("%Y%m%dT%H%M%SZ") %>
+DTSTART:<%= event.starts_at.strftime("%Y%m%dT%H%M%SZ") %>
+DTEND:<%= event.ends_at.presence ? event.ends_at.strftime("%Y%m%dT%H%M%SZ") : (event.starts_at + 1.hour).strftime("%Y%m%dT%H%M%SZ") %>
+SUMMARY:<%= event.name.presence || event.post.topic.title %>
+DESCRIPTION:<%= PrettyText.format_for_email(event.post.excerpt, event.post).html_safe.gsub(/\r?\n/, '\n') %>
+URL:<%= Discourse.base_url %>/t/-/<%= event.post.topic_id %>/<%= event.post.post_number %>
```

Comment:
If every event has a post + topic, we could do `event.post.topic.url(event.post.post_number)` to avoid having to build this up manually.